assignment5.js

```diff
@@ -0,0 +1,24 @@
+var fs = require('fs');
+
+var fileReader = function(fileName) {
+  return new Promise( (resolve, reject) => {
```

Comment:
Write a valid reason in reply for using the arrow function here and not elsewhere.